Failure parsing response from conflict error when sharing file

`Required field missing: .tag ` is hiding the underlying error when parsing the `SharedLinkAlreadyExistsMetadata`.

```

Caused by: com.fasterxml.jackson.core.JsonParseException: Required field missing: .tag  
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 78]  
	at com.dropbox.core.v2.sharing.SharedLinkAlreadyExistsMetadata$Serializer.deserialize(SharedLinkAlreadyExistsMetadata.java:265)  
	at com.dropbox.core.v2.sharing.SharedLinkAlreadyExistsMetadata$Serializer.deserialize(SharedLinkAlreadyExistsMetadata.java:229)  
	at com.dropbox.core.stone.StoneSerializers$NullableSerializer.deserialize(StoneSerializers.java:248)  
	at com.dropbox.core.v2.sharing.CreateSharedLinkWithSettingsError$Serializer.deserialize(CreateSharedLinkWithSettingsError.java:476)  
	at com.dropbox.core.v2.sharing.CreateSharedLinkWithSettingsError$Serializer.deserialize(CreateSharedLinkWithSettingsError.java:401)  
	at com.dropbox.core.ApiErrorResponse$Serializer.deserialize(ApiErrorResponse.java:58)  
	at com.dropbox.core.ApiErrorResponse$Serializer.deserialize(ApiErrorResponse.java:36)  
	at com.dropbox.core.stone.StoneSerializer.deserialize(StoneSerializer.java:66)  
	at com.dropbox.core.DbxWrappedException.fromResponse(DbxWrappedException.java:47)  
	at com.dropbox.core.v2.DbxRawClientV2$1.execute(DbxRawClientV2.java:152)  
```

--- a/dropbox/src/main/java/ch/cyberduck/core/dropbox/DropboxShareFeature.java
+++ b/dropbox/src/main/java/ch/cyberduck/core/dropbox/DropboxShareFeature.java
@@ -25,6 +25,8 @@ import ch.cyberduck.core.exception.ConflictException;
 import ch.cyberduck.core.exception.LoginCanceledException;
 import ch.cyberduck.core.features.Share;
 
+import com.dropbox.core.BadResponseException;
+
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
@@ -69,6 +71,9 @@ public class DropboxShareFeature implements Share<Void, Void> {
                         MessageFormat.format(LocaleFactory.localizedString("{0} URL"),
                                 LocaleFactory.localizedString("Password Share", "Dropbox"))
                 );
+            }
+            catch(BadResponseException e) {
+                throw new ConflictException(e.getMessage(), e);
             }
             catch(DbxException e) {
                 throw new DropboxExceptionMappingService().map(e);

